packages/dropdown/src/Dropdown.tsx

```diff
@@ -32,20 +42,38 @@ export const Dropdown = ({ value, onChange, options, disabled, dataTestId, onBlu
     setIsOpen(false);
   };
 
+  const classNames = cn(
+    styles.container,
+    disabled && styles.disabled,
+    fullWidth && styles.fullWidth,
+  );
+
   return (
     <div
-      className={`${styles.container} ${disabled ? styles.disabled : ''}`}
+      className={classNames}
       onBlur={onBlurHandler}
       onClick={onClickHandler}
       tabIndex={0}
     >
-      <span className={`${styles.value} ${disabled ? 'disabled' : ''}`} >{stateValue?.label}</span>
-      <div className={`${disabled ? styles.caret_disabled : (isOpen ? styles.caret_up : styles.caret_down)}`}></div>
-      <ul className={`${styles.options} ${isOpen ? styles.show : ''}`} data-test-id={dataTestId}>
+      <span className={`${styles.value} ${disabled ? 'disabled' : ''}`}>
+        {stateValue?.label}
+      </span>
+      <div
+        className={`${disabled
```

Comment:
i would suggest a function instead when we have nested ternary operators, it's more readable

Reply:
resolved